.github/workflows/regression-tests.yml

```diff
@@ -56,6 +60,16 @@ jobs:
           name: logs
           path: /tmp/failed/*
 
+      - name: Copy binary on a self hosted runner
+        if: failure()
+        run: |
```

Comment:
Not sure if you "tested" this. I can inject a failure to make sure that it works if you can wait until tomorrow.

Reply:
i tested this.